### ISSUE TITLE
A setting to show or hide content warnings by default; add backend settings persistence

### ISSUE THREAD

**Issue by cinnamon-bun** [fraction/oasis#94]
**What's the problem you want to solved?**
Lots of clicking around hurts my hands, so I'd like to be able to read just by scrolling without having to expand content warnings.

**Is there a solution you'd like to recommend?**
This setting needs to be applied by the backend (in the absence of frontend JS)

1. Add ability to save settings in the backend (not to the SSB log, just a JSON file or something)
2. Add checkbox to Meta screen
3. Apply the setting when generating pages

There are other settings that would be nice like "hide posts containing these words" which could be added on top of this settings infrastructure once we have it.

**Comment by georgeowell** [comment]
This seems pretty important.

**Comment by georgeowell** [comment]
I feel like back-end settings persistence is a whole different issue @cinnamon-bun ?

**Comment by christianbundy** [comment]
Maybe we could just use cookies like we're doing with the theme + i18n?

**Comment by cinnamon-bun** [comment]
Yep, this could be done with cookies.  I renamed the issue to remove "backend settings persistence"
